.gitignore

```diff
@@ -3,3 +3,4 @@
 **/.settings/
 **/.classpath
 **/target/
+**/bin
```

Comment:
is this change necessary ?

Reply:
building with mvn on jdd-9 container saw a few bin folders coming out from the mist: however see no reason not to add this exclusion, no harm.